src/Symfony/Component/Debug/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
-        "symfony/http-kernel": "~2.2",
+        "symfony/http-kernel": "~2.3,>=2.3.24",
```

Comment:
given you changed the inheritance, this needs to become a hard requirement to avoid fatal errors for people using the Debug component standalone